demos/text_to_speech_demo/python/text_to_speech_demo.py

```diff
@@ -78,6 +79,16 @@ def build_argparser():
                        default=None, required=False,
                        type=str)
 
+    args.add_argument("-s_id", "--speaker_id",
+                       help="Ordinal number of the speaker in embeddings array for multi-speaker model.",
```

Comment:
It's worth noting that -1 activates Multi-speaker TTS parameters selection window

Please, update readme listing help message with new args